repo/packages/C/chronos/0/marathon.json

```diff
@@ -28,6 +28,8 @@
       "portIndex": 0,
       "protocol": "HTTP"
     }
-  ]
-
+  ],
+  "labels": {
+    "DCOS_PACKAGE_FRAMEWORK_NAME": "{{chronos/mesos-framework-name}}"
```

Comment:
It has a default value actually so it's taken care of just like {{mesos/master}} and  {{chronos/zk-hosts}}. That said, I realize I need to update the command to expect that it exists. Just updated!